iothub/device/src/Transport/Amqp/AmqpAuthenticationRefresher.cs

```diff
@@ -17,22 +17,22 @@ internal class AmqpAuthenticationRefresher : IAmqpAuthenticationRefresher
         private static readonly string[] s_accessRightsStringArray = AccessRightsHelper.AccessRightsToStringArray(AccessRights.DeviceConnect);
         private readonly Uri _amqpEndpoint;
         private readonly AmqpIotCbsLink _amqpIotCbsLink;
-        private readonly IClientConfiguration _clientConfiguration;
```

Comment:
The amqp changes rea only to refactor the layer to use IotHubConnectionCredentials instead of ClientConfiguration.